server/src/main/java/com/cloud/api/ApiServlet.java

```diff
@@ -136,6 +136,16 @@ public void run() {
         });
     }
 
+    private void ensureSingleQueryParameterValue(Map<String, String[]> params) {
```

Comment:
Can we rename this method since we are not ensuring, but simply warning about it?